apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/Components/CaseDocuments/CaseDocuments.tsx

```diff
@@ -81,6 +83,21 @@ const CaseDocuments: FC<Props> = ({
 
   const isRulingRequired = !workingCase.isCompletedWithoutRuling
 
+  const showRuling: () => boolean = () => {
+    if (isPrisonStaffUser(user)) {
+      return false
+    }
+
+    if (isPrisonAdminUser(user)) {
+      return (
+        isRestrictionCase(workingCase.type) ||
```

Comment:
Do we also consider and `ADMISSION_TO_FACILITY` and `TRAVEL_BAN` here? In the roles rules above we only considered `CUSTODY`

Reply:
True! Thanks. Fixed.